Build fails on Visual Studio 2012 (std::max undefined)

Build of **Horde3D master** fails on Visual Studio 2012.

```
error C2039: 'max' : is not a member of 'std' ..\horde3d\horde3d\source\colladaconverter\daeLibAnimations.h 141 1 ColladaConv
```

The cause seems to be a reorganization of STL in last VS libs.

The fix should be to add the following line to **daeLibAnimations.h**:

```
#include <algorithm>
```

--- a/Horde3D/Source/ColladaConverter/daeLibAnimations.h
+++ b/Horde3D/Source/ColladaConverter/daeLibAnimations.h
@@ -18,6 +18,7 @@
 #include "utils.h"
 #include <string>
 #include <vector>
+#include <algorithm>
 
 using namespace Horde3D;
 

